test/units/test_channel.py

```diff
@@ -0,0 +1,5 @@
+import unittest
```

Comment:
The modern-day standard is to use pytest

Reply:
noted, will update it as I add actual test cases